packages/rum-core/test/utils/apm-server-mock.js

```diff
@@ -82,6 +82,16 @@ class ApmServerMock {
       useMocks ? () => Promise.resolve() : undefined
     )
 
+    this.resetMock = function () {
```

Comment:
This change and the one done in waitForApmServerCalls permit us to use waitForApmServerCalls more than once in the same test if required. Like in https://github.com/elastic/apm-agent-rum-js/pull/1190/files#diff-6c17ea89599249cc59d7b2195a384042aa5c7a12ed67601e5ca04119b8b798c3